diff --git a/src/xrt/auxiliary/tracking/t_calibration.cpp b/src/xrt/auxiliary/tracking/t_calibration.cpp
index 99494b7..db895a7 100644
--- a/src/xrt/auxiliary/tracking/t_calibration.cpp
+++ b/src/xrt/auxiliary/tracking/t_calibration.cpp
@@ -19,6 +19,9 @@
 #include "tracking/t_calibration_opencv.hpp"
 
 #include <opencv2/opencv.hpp>
+#if CV_VERSION_MAJOR >= 5
+#include <opencv2/geometry.hpp>
+#endif
 #include <sys/stat.h>
 #include <utility>
 #include <stdexcept>

